spec/mailers/application_mailer_spec.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailer < ApplicationMailer
+  include ActiveJob::TestHelper
```

Comment:
is this necessary? shouldn't it be incldued in `spec/rails_helper.rb` ?

Reply:
seems to be necessary, I can add it there yes 💪